x-pack/plugins/infra/public/pages/logs/settings/source_configuration_settings.tsx

```diff
@@ -92,110 +83,100 @@ export const LogsSettingsPage = () => {
 
   return (
     <EuiErrorBoundary>
-      <EuiPage>
-        <EuiPageBody
-          className="eui-displayBlock"
-          restrictWidth
-          data-test-subj="sourceConfigurationContent"
-        >
-          <Prompt prompt={isFormDirty ? unsavedFormPromptMessage : undefined} />
-          <EuiPanel paddingSize="l">
-            <NameConfigurationPanel
-              isLoading={isLoading}
-              nameFieldProps={indicesConfigurationProps.name}
-              readOnly={!isWriteable}
-            />
-          </EuiPanel>
-          <EuiSpacer />
-          <EuiPanel paddingSize="l">
-            <IndicesConfigurationPanel
-              isLoading={isLoading}
-              logAliasFieldProps={indicesConfigurationProps.logAlias}
-              readOnly={!isWriteable}
-            />
-          </EuiPanel>
-          <EuiSpacer />
-          <EuiPanel paddingSize="l">
-            <FieldsConfigurationPanel
-              isLoading={isLoading}
-              readOnly={!isWriteable}
-              tiebreakerFieldProps={indicesConfigurationProps.tiebreakerField}
-              timestampFieldProps={indicesConfigurationProps.timestampField}
+      <EuiPage style={{ flex: '1 0 0%' }}>
+        <EuiPageBody data-test-subj="sourceConfigurationContent" restrictWidth>
+          <EuiPageContentBody>
```

Comment:
:information_source: This is in preparation to the new `EuiPageTemplate` component. Unfortunately the corresponding EUI version was not yet merged, so I just used the constituent components to the same effect.